bears/ruby/RubySmellBear.py

```diff
@@ -0,0 +1,37 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.Result import Result
+from coalib.results.SourceRange import SourceRange
+
+
+@linter(executable='reek',
+        use_stdin=True)
+class RubySmellBear:
+    """
+    Detect code smells in Ruby source code.
+    """
+
+    LANGUAGES = "Ruby"
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        return '--format', 'json'
+
+    def process_output(self, output, filename, file):
+        output = json.loads(output) if output else []
```

Comment:
Do we need a JSONDecodeError ?
Does it always give a json output even for corner cases like giving a directory instead of a file, giving a binary file, etc ?
Also, can we always assure that the `output` is the complete output from the process ? Can it hang and have a half completed output ?